README.md

```diff
@@ -108,4 +108,23 @@ Change the install directory using the `PREFIX` environment variable:
   PREFIX=DIR make install
   ```
 
+Debian-specific notes
+---------------------
+
+On Debian testing and unstable, yank can be installed by simply
+running:
+
+  ```
+  sudo apt-get install yank
+  ```
+
+The binary itself is installed as /usr/bin/yank-cli (/usr/bin/yank is
```

Comment:
Change to: The binary is installed as `/usr/bin/yank-cli` due to a naming conflict.

I assume yank users are capable of realizing and creating aliases on their own :)